{
  "uid" : "1ff72c4cd7fa1364",
  "name" : "Verify sizes of images in specialist cards in the grid",
  "fullName" : "tests.specialists_page_test.TestSpecialistCardImages#test_sp_04_02_verify_images_sizes_in_cards",
  "historyId" : "98a366d56cb9c72ff934680675a187fb",
  "time" : {
    "start" : 1728784216377,
    "stop" : 1728784216377,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55c592d5102a <unknown>\n#1 0x55c592a375e0 <unknown>\n#2 0x55c592a2f7eb <unknown>\n#3 0x55c592a1fa09 <unknown>\n#4 0x55c592a21744 <unknown>\n#5 0x55c592a1fcbd <unknown>\n#6 0x55c592a1f52f <unknown>\n#7 0x55c592a1f422 <unknown>\n#8 0x55c592a1d493 <unknown>\n#9 0x55c592a1dada <unknown>\n#10 0x55c592a39f41 <unknown>\n#11 0x55c592acb8c5 <unknown>\n#12 0x55c592aabb22 <unknown>\n#13 0x55c592acad7d <unknown>\n#14 0x55c592aab8c3 <unknown>\n#15 0x55c592a796b3 <unknown>\n#16 0x55c592a7a68e <unknown>\n#17 0x55c592d1ba2b <unknown>\n#18 0x55c592d1f9b1 <unknown>\n#19 0x55c592d08225 <unknown>\n#20 0x55c592d20532 <unknown>\n#21 0x55c592ced38f <unknown>\n#22 0x55c592d3ff28 <unknown>\n#23 0x55c592d400f3 <unknown>\n#24 0x55c592d4fe7c <unknown>\n#25 0x7f0159494ac3 <unknown>",
  "statusTrace" : "driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"94954a667191c2fa4781459fa7cfcd42\")>\n\n    @pytest.fixture()\n    @allure.step(f'Open page: {MainPageLinks.URL_MAIN_PAGE}')\n    def main_page_open(driver):\n>       driver.get(MainPageLinks.URL_MAIN_PAGE)\n\ntests/conftest.py:22: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:353: in get\n    self.execute(Command.GET, {\"url\": url})\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:344: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7faf1f723050>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"unknown error: net::ERR_CONNECTION_REFUSED\\\\n  ...\\\\n#23 0x55c592d400f3 \\\\u003Cunknown>\\\\n#24 0x55c592d4fe7c \\\\u003Cunknown>\\\\n#25 0x7f0159494ac3 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\nE         (Session info: chrome=129.0.6668.89)\nE       Stacktrace:\nE       #0 0x55c592d5102a <unknown>\nE       #1 0x55c592a375e0 <unknown>\nE       #2 0x55c592a2f7eb <unknown>\nE       #3 0x55c592a1fa09 <unknown>\nE       #4 0x55c592a21744 <unknown>\nE       #5 0x55c592a1fcbd <unknown>\nE       #6 0x55c592a1f52f <unknown>\nE       #7 0x55c592a1f422 <unknown>\nE       #8 0x55c592a1d493 <unknown>\nE       #9 0x55c592a1dada <unknown>\nE       #10 0x55c592a39f41 <unknown>\nE       #11 0x55c592acb8c5 <unknown>\nE       #12 0x55c592aabb22 <unknown>\nE       #13 0x55c592acad7d <unknown>\nE       #14 0x55c592aab8c3 <unknown>\nE       #15 0x55c592a796b3 <unknown>\nE       #16 0x55c592a7a68e <unknown>\nE       #17 0x55c592d1ba2b <unknown>\nE       #18 0x55c592d1f9b1 <unknown>\nE       #19 0x55c592d08225 <unknown>\nE       #20 0x55c592d20532 <unknown>\nE       #21 0x55c592ced38f <unknown>\nE       #22 0x55c592d3ff28 <unknown>\nE       #23 0x55c592d400f3 <unknown>\nE       #24 0x55c592d4fe7c <unknown>\nE       #25 0x7f0159494ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py:229: WebDriverException",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "clear_allure_results_folder",
    "time" : {
      "start" : 1728784074935,
      "stop" : 1728784074936,
      "duration" : 1
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "driver",
    "time" : {
      "start" : 1728784216378,
      "stop" : 1728784217150,
      "duration" : 772
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "main_page_open",
    "time" : {
      "start" : 1728784217150,
      "stop" : 1728784217572,
      "duration" : 422
    },
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55c592d5102a <unknown>\n#1 0x55c592a375e0 <unknown>\n#2 0x55c592a2f7eb <unknown>\n#3 0x55c592a1fa09 <unknown>\n#4 0x55c592a21744 <unknown>\n#5 0x55c592a1fcbd <unknown>\n#6 0x55c592a1f52f <unknown>\n#7 0x55c592a1f422 <unknown>\n#8 0x55c592a1d493 <unknown>\n#9 0x55c592a1dada <unknown>\n#10 0x55c592a39f41 <unknown>\n#11 0x55c592acb8c5 <unknown>\n#12 0x55c592aabb22 <unknown>\n#13 0x55c592acad7d <unknown>\n#14 0x55c592aab8c3 <unknown>\n#15 0x55c592a796b3 <unknown>\n#16 0x55c592a7a68e <unknown>\n#17 0x55c592d1ba2b <unknown>\n#18 0x55c592d1f9b1 <unknown>\n#19 0x55c592d08225 <unknown>\n#20 0x55c592d20532 <unknown>\n#21 0x55c592ced38f <unknown>\n#22 0x55c592d3ff28 <unknown>\n#23 0x55c592d400f3 <unknown>\n#24 0x55c592d4fe7c <unknown>\n#25 0x7f0159494ac3 <unknown>\n\n",
    "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
    "steps" : [ {
      "name" : "Open page: https://brainup.site/",
      "time" : {
        "start" : 1728784217151,
        "stop" : 1728784217572,
        "duration" : 421
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55c592d5102a <unknown>\n#1 0x55c592a375e0 <unknown>\n#2 0x55c592a2f7eb <unknown>\n#3 0x55c592a1fa09 <unknown>\n#4 0x55c592a21744 <unknown>\n#5 0x55c592a1fcbd <unknown>\n#6 0x55c592a1f52f <unknown>\n#7 0x55c592a1f422 <unknown>\n#8 0x55c592a1d493 <unknown>\n#9 0x55c592a1dada <unknown>\n#10 0x55c592a39f41 <unknown>\n#11 0x55c592acb8c5 <unknown>\n#12 0x55c592aabb22 <unknown>\n#13 0x55c592acad7d <unknown>\n#14 0x55c592aab8c3 <unknown>\n#15 0x55c592a796b3 <unknown>\n#16 0x55c592a7a68e <unknown>\n#17 0x55c592d1ba2b <unknown>\n#18 0x55c592d1f9b1 <unknown>\n#19 0x55c592d08225 <unknown>\n#20 0x55c592d20532 <unknown>\n#21 0x55c592ced38f <unknown>\n#22 0x55c592d3ff28 <unknown>\n#23 0x55c592d400f3 <unknown>\n#24 0x55c592d4fe7c <unknown>\n#25 0x7f0159494ac3 <unknown>\n\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "driver",
        "value" : "<selenium.webdriver.chrome.webdriver.WebDriver (session=\"94954a667191c2fa4781459fa7cfcd42\")>"
      } ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  } ],
  "afterStages" : [ {
    "name" : "driver::0",
    "time" : {
      "start" : 1728784217604,
      "stop" : 1728784217666,
      "duration" : 62
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "Test Specialists Page"
  }, {
    "name" : "parentSuite",
    "value" : "tests"
  }, {
    "name" : "suite",
    "value" : "specialists_page_test"
  }, {
    "name" : "subSuite",
    "value" : "TestSpecialistsPage"
  }, {
    "name" : "host",
    "value" : "fv-az891-551"
  }, {
    "name" : "thread",
    "value" : "2367-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.specialists_page_test"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 4,
        "skipped" : 0,
        "passed" : 27,
        "unknown" : 0,
        "total" : 31
      },
      "items" : [ {
        "uid" : "ff582e387b7f4064",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/60//#testresult/ff582e387b7f4064",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x5648a916102a <unknown>\n#1 0x5648a8e475e0 <unknown>\n#2 0x5648a8e3f7eb <unknown>\n#3 0x5648a8e2fa09 <unknown>\n#4 0x5648a8e31744 <unknown>\n#5 0x5648a8e2fcbd <unknown>\n#6 0x5648a8e2f52f <unknown>\n#7 0x5648a8e2f422 <unknown>\n#8 0x5648a8e2d493 <unknown>\n#9 0x5648a8e2dada <unknown>\n#10 0x5648a8e49f41 <unknown>\n#11 0x5648a8edb8c5 <unknown>\n#12 0x5648a8ebbb22 <unknown>\n#13 0x5648a8edad7d <unknown>\n#14 0x5648a8ebb8c3 <unknown>\n#15 0x5648a8e896b3 <unknown>\n#16 0x5648a8e8a68e <unknown>\n#17 0x5648a912ba2b <unknown>\n#18 0x5648a912f9b1 <unknown>\n#19 0x5648a9118225 <unknown>\n#20 0x5648a9130532 <unknown>\n#21 0x5648a90fd38f <unknown>\n#22 0x5648a914ff28 <unknown>\n#23 0x5648a91500f3 <unknown>\n#24 0x5648a915fe7c <unknown>\n#25 0x7f73c4a94ac3 <unknown>",
        "time" : {
          "start" : 1728697330308,
          "stop" : 1728697330308,
          "duration" : 0
        }
      }, {
        "uid" : "41950eec56ce105c",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/59//#testresult/41950eec56ce105c",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55b218fa402a <unknown>\n#1 0x55b218c8a5e0 <unknown>\n#2 0x55b218c827eb <unknown>\n#3 0x55b218c72a09 <unknown>\n#4 0x55b218c74744 <unknown>\n#5 0x55b218c72cbd <unknown>\n#6 0x55b218c7252f <unknown>\n#7 0x55b218c72422 <unknown>\n#8 0x55b218c70493 <unknown>\n#9 0x55b218c70ada <unknown>\n#10 0x55b218c8cf41 <unknown>\n#11 0x55b218d1e8c5 <unknown>\n#12 0x55b218cfeb22 <unknown>\n#13 0x55b218d1dd7d <unknown>\n#14 0x55b218cfe8c3 <unknown>\n#15 0x55b218ccc6b3 <unknown>\n#16 0x55b218ccd68e <unknown>\n#17 0x55b218f6ea2b <unknown>\n#18 0x55b218f729b1 <unknown>\n#19 0x55b218f5b225 <unknown>\n#20 0x55b218f73532 <unknown>\n#21 0x55b218f4038f <unknown>\n#22 0x55b218f92f28 <unknown>\n#23 0x55b218f930f3 <unknown>\n#24 0x55b218fa2e7c <unknown>\n#25 0x7ff13c094ac3 <unknown>",
        "time" : {
          "start" : 1728611021542,
          "stop" : 1728611021542,
          "duration" : 0
        }
      }, {
        "uid" : "33bec92c096096f9",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/58//#testresult/33bec92c096096f9",
        "status" : "passed",
        "time" : {
          "start" : 1728525035585,
          "stop" : 1728525035740,
          "duration" : 155
        }
      }, {
        "uid" : "592dfeefea094494",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/57//#testresult/592dfeefea094494",
        "status" : "passed",
        "time" : {
          "start" : 1728438706762,
          "stop" : 1728438706919,
          "duration" : 157
        }
      }, {
        "uid" : "9f0388a24e255f57",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/56//#testresult/9f0388a24e255f57",
        "status" : "passed",
        "time" : {
          "start" : 1728352295665,
          "stop" : 1728352295837,
          "duration" : 172
        }
      }, {
        "uid" : "6432fda7fb2b1151",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/55//#testresult/6432fda7fb2b1151",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find elements by locator ('xpath', \"//img[@class='rounded-full']\")",
        "time" : {
          "start" : 1728266868721,
          "stop" : 1728266888974,
          "duration" : 20253
        }
      }, {
        "uid" : "39d84478454e6fa0",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/54//#testresult/39d84478454e6fa0",
        "status" : "passed",
        "time" : {
          "start" : 1728179838135,
          "stop" : 1728179838295,
          "duration" : 160
        }
      }, {
        "uid" : "5ca6856678fdb749",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/53//#testresult/5ca6856678fdb749",
        "status" : "passed",
        "time" : {
          "start" : 1728093085094,
          "stop" : 1728093085253,
          "duration" : 159
        }
      }, {
        "uid" : "bed926f6d6ec535",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/52//#testresult/bed926f6d6ec535",
        "status" : "passed",
        "time" : {
          "start" : 1728006788603,
          "stop" : 1728006788761,
          "duration" : 158
        }
      }, {
        "uid" : "bab6b3713f37d4aa",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/51//#testresult/bab6b3713f37d4aa",
        "status" : "passed",
        "time" : {
          "start" : 1727920450408,
          "stop" : 1727920450571,
          "duration" : 163
        }
      }, {
        "uid" : "8cbe3cca817f3422",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/50//#testresult/8cbe3cca817f3422",
        "status" : "passed",
        "time" : {
          "start" : 1727833869883,
          "stop" : 1727833870046,
          "duration" : 163
        }
      }, {
        "uid" : "9c3584e6c7623138",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/49//#testresult/9c3584e6c7623138",
        "status" : "passed",
        "time" : {
          "start" : 1727747995475,
          "stop" : 1727747995636,
          "duration" : 161
        }
      }, {
        "uid" : "178dd38e6e3de77a",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/48//#testresult/178dd38e6e3de77a",
        "status" : "passed",
        "time" : {
          "start" : 1727661275764,
          "stop" : 1727661275924,
          "duration" : 160
        }
      }, {
        "uid" : "3b5c36129cb4b5d",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/47//#testresult/3b5c36129cb4b5d",
        "status" : "passed",
        "time" : {
          "start" : 1727575074017,
          "stop" : 1727575074176,
          "duration" : 159
        }
      }, {
        "uid" : "72ce0648a3553c73",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/46//#testresult/72ce0648a3553c73",
        "status" : "passed",
        "time" : {
          "start" : 1727488336114,
          "stop" : 1727488336270,
          "duration" : 156
        }
      }, {
        "uid" : "38bd04f4b2f1d1df",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/45//#testresult/38bd04f4b2f1d1df",
        "status" : "passed",
        "time" : {
          "start" : 1727402057047,
          "stop" : 1727402057213,
          "duration" : 166
        }
      }, {
        "uid" : "a0d6ff59f85b1504",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/44//#testresult/a0d6ff59f85b1504",
        "status" : "passed",
        "time" : {
          "start" : 1727315567525,
          "stop" : 1727315567681,
          "duration" : 156
        }
      }, {
        "uid" : "d495553c94fa89f",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/43//#testresult/d495553c94fa89f",
        "status" : "passed",
        "time" : {
          "start" : 1727229173448,
          "stop" : 1727229173614,
          "duration" : 166
        }
      }, {
        "uid" : "9fbc4f8a6597b61c",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/42//#testresult/9fbc4f8a6597b61c",
        "status" : "passed",
        "time" : {
          "start" : 1727142701097,
          "stop" : 1727142701261,
          "duration" : 164
        }
      }, {
        "uid" : "619f64b468e46d05",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/41//#testresult/619f64b468e46d05",
        "status" : "passed",
        "time" : {
          "start" : 1727056219338,
          "stop" : 1727056219499,
          "duration" : 161
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "1ff72c4cd7fa1364.json",
  "parameterValues" : [ ]
}